### PR TITLE
MTL-3515 Remove Visa SRC from Demo page

### PR DESCRIPTION
**Pull request by jgalvanbam**
1.- Remove Visa SRC from Demo  page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,7 @@ pip install -r requirements.txt
 
 ### Set up Environment Variables
 
-We are using two test accounts in this demo app. One is on Production, the other is on Sandbox. All services,
-except Visa Checkout can be tested on Production. Visa Checkout must be tested on Sandbox.
+We are using one test account on Pruduction in this demo app. All services can be tested on Production.
```
Comment (rb-bam) [inline README.md:61]: Production spelling error here.

File: server/app/server.py [hunk 2]
```diff
@@ -44,8 +43,8 @@ def version():
 
 @app.route('/')
 def get_landing_page():
-    visa_checkout_api_key = settings.sandbox_visa_checkout_api_key
-    return render_template('index.html', api_key=visa_checkout_api_key)
+    secret_key = settings.secret_key
```
Comment (rb-bam) [inline server/app/server.py:46]: I don't understand why this is changed to pass a different settings value. Shouldn't this be changed to not pass in a second parameter?

**Review by rb-bam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jgalvanbam) [reply README.md]: Typo error fixed

File: server/app/server.py (hunk 2 above)
Comment (jgalvanbam) [reply server/app/server.py]: Variable removed and set to pass nothing as a second parameter.

**Review by rb-bam — APPROVED** [review]

**Pull request merged**